src/kv/MemDB.cc

```diff
@@ -227,7 +227,7 @@ void MemDB::MDBTransactionImpl::rmkey(const string &prefix,
   dtrace << __func__ << " " << prefix << " " << k << dendl;
   ops.push_back(make_pair(DELETE,
                           std::make_pair(std::make_pair(prefix, k),
-                          NULL)));
+                          bufferlist() )));
```

Comment:
@tchaikov 
Did that on purpose, to separate the function end from the other )s.
But it does not conform the standard.